src/helm/proxy/models.py

```diff
@@ -374,6 +374,31 @@ def engine(self) -> str:
         "JavaScript subset of The Stack (v1.1).",
         tags=[CODE_MODEL_TAG],
     ),
+    # Local HuggingFace models
+    Model(
+        group="local",
+        creator_organization="Meta",
+        name="local/llama-7b",
+        display_name="LLaMA (7B)",
+        description="", # TODO add description
+        tags=[TEXT_MODEL_TAG, FULL_FUNCTIONALITY_TEXT_MODEL_TAG]
```

Comment:
Could you add a short comment explaining how to run these models i.e. "copy the model files into "./huggingface_models/llama-6b" and if the Hugging Face conversion script is needed